runner/src/lib.rs

```diff
@@ -256,11 +309,39 @@ fn bind_functions(module: &Module, execution_engine: &ExecutionEngine) -> Result
     bind!(__quantum__rt__array_concatenate);
     bind!(__quantum__rt__array_copy);
     bind!(__quantum__rt__array_create_1d);
+
+    // Legacy output methods
     bind!(__quantum__rt__array_end_record_output);
```

Comment:
Just to double check, the way these are linked in now means only the deprecation error message will show up, and that is intentional, correct? That is a bit different than the other functions that link in the old functionality if needed, such that old code that doesn't use aggregates will seem to work fine but then anything with aggregates will print the deprecation warning. Seems like that could be a confusing experience, if I'm understanding it correctly.